### DISCUSSION TITLE
Fedora 40 Server Edition install worked but issues adding existing game servers

### DISCUSSION THREAD

**Discussion by demianph** [BlueSquare23/web-lgsm#22] [Q&A] [answered]
Installed to Fedora 40 Server Edition.
It's super easy, just did the same install commands but do packages via DNF (python3 already installed in OS by default just had to add python3-pip).
Created a user to login to the webui and get access just fine  (After modifying main.conf to bind to accessible IP) and was able to login.
I have a filesystems /gameservers where I create my lgsm accounts/directories
All LGSM users and directories share a group of gameservers (all file group ownership is gameservers)
Attempted to add an existing LGSM for satisfactory
Install path is /gameservers/sfserver
script name is sfserver
server user account is sfserver
When I attempt to add I get a red popup saying "Script file does not exist."
I even put the wlgsm user in the same common group with the game servers and made sure it has sudo level access.
Ideas?
Also how do I set this up so if I use the UI install links for new servers it will install to my location?

**Comment by BlueSquare23** [comment, marked as answer]
Cool to see someone get it up and running on Fedora! I'll have to build out the install.sh script to support that at some point.

If you are absolutely sure that the `/gameservers/sfserver/sfserver` script file is there and the web-lgsm system user has access to read and execute (r-x) it, then it could be a bug. Good new is I just re-wrote a lot of that code, so if it is a bug will probably be squashed in 1.8. Bad news is that doesn't do anything for you right now.

But as for right now what you can do to get the game server added to the web interface anyways is just comment out those lines that check for the presence of the script path. If you're on v1.7.0 it should be lines #625 to 628 in the `app/views.py` file. Then restart the web-lgsm and then you should be able to add your game server! 

### Comment out these lines:
https://github.com/BlueSquare23/web-lgsm/blob/1d19c0f770135db63ba0bcfe0f5240cf2c80b785/app/views.py#L625-L628

I think that should work for you but lmk if you hit any other problems!

**Reply by demianph** [reply]
Decided to uninstall both web-lgsm and the satisfactory server and re-do the install to make sure I didn't have a permission issue I missed.

Somehow completely broke the web server when it re-installed, get 
"Internal Server Error
The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application."
running back through process.
Thanks for the help, when I unbreak the webserver will add per your instructions.

Found I missed a step from the install (risk when operating by hand) with the secret file.
Good to go, thank you again.

**Reply by demianph** [reply]
As for the script permissions.
-rwxr-xr-x.  1 sfserver gameservers 18433 Sep 12 13:34 sfserver

[sfserver@3550m4 ~]$ id wlgsm
uid=1002(wlgsm) gid=1003(gameservers) groups=1003(gameservers)

So must just be selinux preventing the add as I have not dug into that aspect yet.
